compose/cli/errors.py

```diff
@@ -55,6 +56,8 @@ def handle_connection_errors(client):
             if is_ubuntu():
                 exit_with_error(docker_not_found_ubuntu)
             exit_with_error(docker_not_found_generic)
+        if is_docker_for_mac_installed():
+            exit_with_error(conn_error_docker_for_mac)
```

Comment:
Should this go above the first `if which docker ` line? Or does docker for mac not include a docker binary in the path?